applications/IgaApplication/tests/cpp_tests/test_creation_utility.h

```diff
@@ -130,6 +131,52 @@ namespace TestCreationUtility
         return result_geometries(0);
     }
 
+
+    inline typename Geometry<NodeType>::Pointer GetQuadraturePointGeometryOnCurve(
+        ModelPart& rModelPart, SizeType PolynomialDegree, IntegrationPoint<3> IntegrationPoint)
+    {
```

Comment:
```suggestion
    inline typename Geometry<NodeType>::Pointer GetQuadraturePointGeometryOnCurve(
        ModelPart& rModelPart,
        SizeType PolynomialDegree,
        IntegrationPoint<3> IntegrationPoint)
    {
```
Note that we use one line per argument in multiple argument functions. Besides, shouldn't `IntegrationPoint` be passed by reference?